server/views/components/customDisplay/crm7/crm7Disbursements.njk

```diff
@@ -33,18 +35,18 @@
 
     {# Render "Disbursements Total" subsection #}
     {% set fields = [
-        { label: "Total Net", value: data.totals.net | formatCurrency},
-        { label: "Total VAT", value: data.totals.vat | formatCurrency},
+        { label: "Total Net", value: data.totals.net | formatCurrency, class: "govuk-brand-colour govuk-!-font-weight-bold"},
+        { label: "Total VAT", value: data.totals.vat | formatCurrency, class: "govuk-brand-colour govuk-!-font-weight-bold"},
         { label: "Grand Total", value: data.totals.total | formatCurrency, class: "govuk-brand-colour govuk-!-font-weight-bold"}
 
     ] %}
-    {{ formSubsection('Disbursements total', fields) }}
+    {{ formSubsection('Disbursements Total', fields) }}
 
 
     {# Render "Office Use Only" subsection #}
     {% set fields = [
-        { label: "Total Net", value: data.officeUse.net | formatCurrency},
-        { label: "Total VAT", value: data.officeUse.vat | formatCurrency},
+        { label: "Total Net", value: data.officeUse.net | formatCurrency, class: "govuk-brand-colour govuk-!-font-weight-bold"},
```

Comment:
Total Net, Total VAT, Grand Total rows should be black, not blue.

Reply:
Its fixed and ready for re-review.